test/ffmpeg-qsv/vpp/csc.py

```diff
@@ -40,7 +40,8 @@ def test_default(case, csc):
 
   check_metric(
     # if user specified metric, then use it.  Otherwise, use ssim metric with perfect score
-    metric = params.get("metric", dict(type = "ssim", miny = 1.0, minu = 1.0, minv = 1.0)),
+    metric = params.get("metric", dict(type = "ssim", miny = 0.99, minu = 0.99, minv = 0.99)
+      if params.get("reference") else dict(type = "ssim", miny = 1.0, minu = 1.0, minv = 1.0)),
```

Comment:
I disagree.  If user specifies a reference, we should assume they want a perfect score by default.  Otherwise, user should set the min params explicitly in the config to override the default.

Reply:
ok, let's set the min ssim in config case, so close this PR.